__tests__/app.test.js

```diff
@@ -153,10 +153,9 @@ describe("GET reviews ", () => {
       .get("/api/reviews")
       .expect(200)
       .then(({ body }) => {
-        console.log(body);
-        expect(body.reviews.length > 0).toBe(true);
-        expect(Array.isArray(body.reviews)).toBe(true);
-        body.reviews.forEach((review) => {
+        expect(body.review.length > 0).toBe(true);
```

Comment:
This expect statement doesn't tell me anything useful, it just tells me it's not empty. I would recommend you amend this expect statement to a length test where it tells me exactly how many articles have been returned, this would give me increased confidence that the test api call to /api/reviews has worked and returned what I expect to see